tools/gluten-it/src/main/scala/io/glutenproject/integration/tpc/TpcSuite.scala

```diff
@@ -76,6 +77,16 @@ abstract class TpcSuite(
     sessionSwitcher.defaultConf().set("spark.default.parallelism", "1")
   }
 
+  if (enableCeleborn) {
+    sessionSwitcher.defaultConf().set("spark.shuffle.manager",
+      "org.apache.spark.shuffle.celeborn.CelebornShuffleManager")
+    sessionSwitcher.defaultConf().set("spark.celeborn.shuffle.writer", "hash")
+    sessionSwitcher.defaultConf().set("spark.celeborn.push.replicate.enabled", "false")
+    sessionSwitcher.defaultConf().set("spark.shuffle.service.enabled", "false")
+    sessionSwitcher.defaultConf().set("spark.sql.adaptive.localShuffleReader.enabled", "false")
+    sessionSwitcher.defaultConf().set("spark.dynamicAllocation.enabled", "false")
+  }
```

Comment:
`sessionSwitcher.defaultConf()` will be applied to both baseline session (Vanilla Spark) and test session (Gluten). Do these config options make Celeborn work on Vanilla Spark correctly? Did you check on that?

Reply:
@zhztheplayer Celeborn configurations have been added to test session.